src/api/fetch.ts

```diff
@@ -0,0 +1,21 @@
+import { BACKEND_URL } from "./constants";
+
+export const executeReq = (query: string) => {
+	return fetch(BACKEND_URL, {
+		method: "POST",
+		headers: {
+			"Content-Type": "application/json",
+			Accept: "application/json",
+		},
+		body: JSON.stringify({
+			query,
+		}),
+	})
```

Comment:
Can you please explain me what we are doing here?

Reply:
We are trying to call the backend API here with the body being passed as a query param. Similar to what we do in axios.